python/lsst/sims/photUtils/PhotometricParameters.py

```diff
@@ -29,42 +29,44 @@ class DefaultPhotometricParameters(object):
 
     bandpassNames = ['u', 'g', 'r', 'i', 'z', 'y', 'any']
```

Comment:
Should we go ahead and set `bandpassNames` from `list(exptime.keys())` after assigning `exptime`.  I'm not 100% sure that we actually need to keep `bandpassNames` as a part of this class, anyway